src/application/F3DOptionsParser.cxx

```diff
@@ -378,13 +344,30 @@ void ConfigurationOptions::PrintHelp(cxxopts::Options& cxxOptions)
 //----------------------------------------------------------------------------
 void ConfigurationOptions::PrintVersion()
 {
-  f3d::engine::printVersion();
+  f3d::log::info(F3D::AppName + " " + F3D::AppVersion + "\n");
+  f3d::log::info(F3D::AppTitle);
+
+  auto libInfo = f3d::engine::getLibInfo2();
+  std::vector<std::string> orderedInfo = {"Version", "Build date", "Build system", "Compiler", "Raytracing module", "Exodus module", "OpenCASCADE module", "Assimp module", "Alembic module", "VTK version", "License", "Authors" };
+
+  f3d::log::setUseColoring(false);
+  for (std::string key : orderedInfo)
+  {
+    f3d::log::info(key + ": " + libInfo[key] + ".");
+  }
+  f3d::log::info(libInfo["Copyright_previous"]);
+  f3d::log::info(libInfo["Copyright"]);
+  f3d::log::setUseColoring(true);
+  f3d::log::waitForUser();
```

Comment:
@Meakk : That a first version. I'm not really convinced though.

Reply:
It would be clearer to call `f3d::log::info(libInfo["XXX"]);` explicitly instead of building a vector and doing a loop imho.

Reply:
that would be more line but ok. Still not great, you need to know the "Keys".